src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -213,7 +213,7 @@ public function load(array $configs, ContainerBuilder $container)
             }
 
             if (!\extension_loaded('intl') && !\defined('PHPUNIT_COMPOSER_INSTALL')) {
-                @trigger_error('Please install the "intl" PHP extension for best performance.', E_USER_DEPRECATED);
+                deprecated('', '', 'Please install the "intl" PHP extension for best performance.');
```

Comment:
I am not sure, but what about adding the same method “suggestion()” without the vendor and version parameter?

This reads more natural, and fixes the concerns of @javiereguiluz

Composer is also “suggesting” things which you can but not need to install/do

Reply:
I think we need to stick to the strict minimum. This does the job, which means we don't need more semantics (aka complexity in this case.)

Reply:
Still the old function name